### DISCUSSION TITLE
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='drive.google.com', port=443)

### DISCUSSION THREAD

**Discussion by zyc01** [mikel-brostrom/yolo_tracking#574] [Q&A]
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='drive.google.com', port=443): Max retries exceeded with url: /uc?id=1Kkx2zW89jq_NETu4u42CFZTMVD5Hwm6e (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000027ECE71F8B0>: Failed to establish a new connection: [WinError 10060]

how do you slove this problem?

**Comment by wuchengyu123** [comment]
I also encountered this problem. And i also had no idea.😂
